shiny/render/_render.py

```diff
@@ -115,8 +115,8 @@ async def PlotTransformer(
     _fn: ValueFn[object],
     *,
     alt: Optional[str] = None,
-    width: Optional[float] = None,
-    height: Optional[float] = None,
+    width: Optional[float] | MISSING_TYPE = MISSING,
+    height: Optional[float] | MISSING_TYPE = MISSING,
```

Comment:
Is `None` still a valid value?

If so, the intent would be clearer with `float | None | MISSING_TYPE`. Also, do the docs explain what `None` means?

```suggestion
    width: float | None | MISSING_TYPE = MISSING,
    height: float | None  | MISSING_TYPE = MISSING,
```

If not, take out the `Optional`. 

```suggestion
    width: float | MISSING_TYPE = MISSING,
    height: float | MISSING_TYPE = MISSING,
```

Reply:
I think the difference only matters when you're using the implicit output. `None` means the implicit output shouldn't have any width/height attribute, `MISSING` means the implicit output should use its default width/height attribute. I can't really document that until we launch express mode, maybe for now I just say None or MISSING mean the same?